### ISSUE TITLE
boost popular entities in live search results

### ISSUE THREAD

**Issue by maxlath** [inventaire/inventaire#101]
In the current implementation, searching `Victor Hugo` returns many homonyms but not [`Q535`](https://inventaire.io/entity/wd:Q535).
This could be fixed by using [ElasticSearch `function_score` feature](https://www.elastic.co/guide/en/elasticsearch/guide/current/boosting-by-popularity.html) but first requires to generate a popularity score when indexing entities. Those scores could depend on the entity type:

works: number of instances in users' inventories
series/humans: cumulated works score

**Comment by maxlath** [comment]
After [working on making entities deduplication easier](https://wiki.inventaire.io/wiki/August-2017) it is fortunately less critical, but still very much needed

![harry potter](https://user-images.githubusercontent.com/1596934/30346218-6219e1be-9808-11e7-82bb-b076ef3a3e4b.png)

**Comment by maxlath** [comment]
another example: searching Karl Marx should return wd:Q9061 as first result ([currently 7th result](https://inventaire.io/api/entities?action=search-type&type=humans&search=karl%20marx))

**Comment by jum-s** [comment]
"Karl Marx should return wd:Q9061 as first result" is hard to achieve because other people are also called exactly Karl Marx (ie. "Q1732312", "Q16062483")

What about a looser spec:
-  Q9061,  Q1732312 & Q16062483 (the multiple "Karl Marx"es) should have a score above Q214986 ("Heinrich Marx") & Q27867982 ("Edgar Marx") => Meaning result should favor full "match_phrase"

**Comment by jum-s** [comment]
Then, once we have all the karl marx(es) as top result, we could try something more exotic: label translations occurrences popularity. Put it in another way: result based on how many occurrences of the query appear in the labels.
- Q17521882 has 3 Karl Marx in the labels (:fr, :es, :nl),  Q1732312 has 4 (:de, :fr, :nl, :en), Q9061 has more than 150 !
One bias might be that it favor worldwilde characters (who have lots of translation) as opposition to local people.

**Comment by maxlath** [comment]
or we re-index those entities with [their popularity score](https://inventaire.io/api/entities?action=popularity&uris=wd:Q1732312|wd:Q27867982|wd:Q17521882|wd:Q16062483|wd:Q214986|wd:Q1732308|wd:Q9061|wd:Q259334|wd:Q1732309|wd:Q20088846|wd:Q13816202|wd:Q1622032|wd:Q19841721|wd:Q61412|wd:Q14816057|wd:Q882538|wd:Q21520064|wd:Q238302|wd:Q448860|wd:Q14815775) and give a boost accordingly

**Comment by jum-s** [comment]
i dont know what to feel about [this test](https://github.com/inventaire/inventaire/compare/101-boost-entities-in-livesearch#diff-6ef2a81d053ebe6fe72d8866f96299faR51) i just wrote.
one can say its mocking too much the current behavior (aka: creating an edition with a work to sort by popularity), but maybe in the future this manner to calculate the popularity score will desappear right? so can i assign an entity with a popularity key directly ? ie. 
```
    createWork { labels: { fr: 'foo' } }
    .then (work)-> createWork.popularity = 2
```
and check the search result ?

**Comment by maxlath** [comment]
fixed by #158
